src/test/kbaseServiceMock.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * A testing http server for simulating KBase service endpoints.
+ *
+ * When testing components, apis, or other elements of Europa which depend upon contacting
+ * external resources via http, the recommended approach is to avoid mocking
+ * local code like api clients, but rather to provide an actual api endpoint.
+ *
+ * This implementation uses the "mirage" library, which intercepts fetch
+ * requests.
+ *
+ * One implication of this is that ALL fetch requests will be run against the
+ * mock server. Any tests which inadvertently rely upon contacting CI services
+ * will fail unless support is added herein.
+ *
+ * Note too that the host origin is 'http://localhost'. See
+ * `src/common/api/index.ts` which defines baseUrl for the test environment.
+ */
+import { MockResponseInit } from 'jest-fetch-mock/types';
+import {
+  STATUS_1,
+  VIEW_ASSEMBLY_BRIEF_INFO_1,
+} from './data/narrative_method_store';
+import { KBASEUITEST_PROFILE } from './data/user_profile';
+import { jsonrpc11_resultResponse } from './jsonrpc11';
+
+/**
+ * Creates response for fetch mock supplied by jest-fetch-mock.
+ *
+ * Most data used for responses is located in the "data" directory.
+ *
+ */
+export function makeKBaseServices() {
```

Comment:
Why "make" and not "mock"?

Reply:
just because it is making an instance of kbase service mock endpoints and method implementations. It is making a mock...